Jellyfin.Data/Entities/MediaSegment.cs

```diff
@@ -0,0 +1,52 @@
+using System;
+using System.Text.Json.Serialization;
+
+namespace Jellyfin.Data.Entities;
+
+/// <summary>
+///     An entity representing the metadata for a group of trickplay tiles.
+/// </summary>
+public class MediaSegment
+{
+    /// <summary>
+    ///     Gets or sets the id of the media segment.
+    /// </summary>
+    /// <remarks>
+    /// Required.
+    /// </remarks>
```

Comment:
```suggestion
```

These remarks are unneeded

Reply:
Note: might as well remove them on the trickplay comments too